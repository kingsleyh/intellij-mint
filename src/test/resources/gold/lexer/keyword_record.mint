record A {
  name : String,
  list : Array(Item)
}