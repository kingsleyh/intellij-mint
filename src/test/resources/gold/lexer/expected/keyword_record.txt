record ('record')
WHITE_SPACE (' ')
IDENT ('A')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n  ')
IDENT ('name')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
IDENT ('String')
, (',')
WHITE_SPACE ('\n  ')
IDENT ('list')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
IDENT ('Array')
( ('(')
IDENT ('Item')
) (')')
WHITE_SPACE ('\n')
} ('}')